tools/cmd/docs/transformations.go

```diff
@@ -174,43 +181,36 @@ func buildExamples(cmd *cobra.Command, examples map[string][]metadatatypes.Examp
 				sb.WriteString("\n      :tabid: ")
 				sb.WriteString(version)
 				sb.WriteString("_")
-				if ex.Source == "-" {
-					sb.WriteString("default")
-				} else {
-					sb.WriteString(strings.ToLower(strings.ReplaceAll(ex.Source, " ", "_")))
-				}
+				sb.WriteString(source)
 				sb.WriteString("\n\n")
 			}
 
 			if ex.Value != "" {
 				if tabs {
 					sb.WriteString("      ")
 				}
-				sb.WriteString("Create the file below and save it as ``payload.json``\n\n")
+				sb.WriteString(ex.Description + "\n\n")
 
 				if tabs {
 					sb.WriteString("      ")
 				}
-				sb.WriteString(".. code-block::\n\n")
-				if ex.Description != "" {
-					if tabs {
-						sb.WriteString("      ")
-					}
-					sb.WriteString("   # " + ex.Description + "\n")
-				}
+				sb.WriteString("Create the file below and save it as ``payload.json``\n\n")
 
-				lines := strings.Split(ex.Value, "\n")
-				for _, line := range lines {
-					if tabs {
-						sb.WriteString("      ")
-					}
+				includePayloadFileName := fmt.Sprintf("%s-%s-%s-payload.json", strings.ReplaceAll(cmd.CommandPath(), " ", "-"), version, source)
+				const permissions = 0600
+				err := os.WriteFile(fmt.Sprintf("%s/%s", includesPath, includePayloadFileName), []byte(ex.Value), permissions)
+				if err != nil {
+					return "", fmt.Errorf("failed to write payload file %s: %w", includePayloadFileName, err)
+				}
 
-					sb.WriteString("   " + line + "\n")
+				if tabs {
+					sb.WriteString("      ")
 				}
+				sb.WriteString(fmt.Sprintf("   .. literalinclude:: /includes/%s\n", includePayloadFileName))
```

Comment:
we likely need to amend this as well if we add a new workflow that adds a subdirectory for these examples under includes
```suggestion
				sb.WriteString(fmt.Sprintf("   .. literalinclude:: /includes/atlas-cli-examples/%s\n", includePayloadFileName))
```